Content.Server/Hands/Systems/HandsSystem.cs

```diff
@@ -95,6 +101,25 @@ private void OnDisarmed(EntityUid uid, HandsComponent component, DisarmedEvent a
             if (!_handsSystem.TryDrop(uid, component.ActiveHand!, null, checkActionBlocker: false))
                 return;
 
+            if (!TryComp<CombatModeComponent>(args.Target, out var combatMode))
+            {
+                return;
+            }
+
+            var filterOther = Filter.PvsExcept(args.Source, entityManager: EntityManager);
+
+            var msgOther = Loc.GetString(
+                    "disarm-action-popup-message-other-clients",
+                    ("performerName", Identity.Entity(args.Source, EntityManager)),
+                    ("targetName", Identity.Entity(args.Target, EntityManager)));
+
+            var msgUser = Loc.GetString("disarm-action-popup-message-cursor", ("targetName", Identity.Entity(args.Target, EntityManager)));
+
+            _popupSystem.PopupEntity(msgOther, args.Source, filterOther, true);
+            _popupSystem.PopupEntity(msgUser, args.Target, args.Source);
+
+            _audio.PlayPvs(combatMode.DisarmSuccessSound, args.Source, AudioParams.Default.WithVariation(0.025f).WithVolume(5f));
+
             args.Handled = true; // no shove/stun.
         }
```

Comment:
Hands shouldn't worry about combat, just check if the event that's there is handled.